### PR TITLE
Add react-native-header-scroll-view to Community-developed Navigators and Libraries

### PR DESCRIPTION
**Pull request by thomaswangio**
Love this little library from @jonsamp and adding it to `Community-developed Navigators and Libraries` after feedback from @satya164 on Twitter: https://twitter.com/satya164/status/1179174667552210944

### CODE REVIEW

File: docs/community-libraries-and-navigators.md [hunk 1]
```diff
@@ -59,3 +59,20 @@ Provides simple HOCs that map react-navigation props to your screen components d
...
 
 [github.com/vonovak/react-navigation-props-mapper](https://github.com/vonovak/react-navigation-props-mapper)
+
+## react-native-header-scroll-view
+
+This component addresses the most requested feature to React Navigation – [iOS large header and grow/shrink on scroll](https://react-navigation.canny.io/feature-requests/p/ios-11-large-header-and-growshrink-on-scroll), made by [@jonsamp](https://github.com/jonsamp). It is different than the actual iOS large header behavior, but does the header title shrinking well. It can be used with React Navigation by using the following code in your component:
+
+```
+  static navigationOptions = {
+    headerMode: "none",
```
Comment (satya164) [inline docs/community-libraries-and-navigators.md]: `headerMode` is an option in `createStackNavigator`'s config, not in `navigationOptions`. I'd document both, `headerMode: 'none'` to disable globally, and `header: null` to disable header for one screen.

File: docs/community-libraries-and-navigators.md [hunk 2]
```diff
@@ -59,3 +59,20 @@ Provides simple HOCs that map react-navigation props to your screen components d
 #### Links
 
 [github.com/vonovak/react-navigation-props-mapper](https://github.com/vonovak/react-navigation-props-mapper)
+
+## react-native-header-scroll-view
+
+This component addresses the most requested feature to React Navigation – [iOS large header and grow/shrink on scroll](https://react-navigation.canny.io/feature-requests/p/ios-11-large-header-and-growshrink-on-scroll), made by [@jonsamp](https://github.com/jonsamp). It is different than the actual iOS large header behavior, but does the header title shrinking well. It can be used with React Navigation by using the following code in your component:
```
Comment (satya164) [inline docs/community-libraries-and-navigators.md]:
```suggestion
This component implements [iOS large header with grow/shrink on scroll](https://react-navigation.canny.io/feature-requests/p/ios-11-large-header-and-growshrink-on-scroll), made by [@jonsamp](https://github.com/jonsamp). Note that it doesn't handle header animation between screens, it only handles animating the header title on scroll.

To use this component for header in React Navigation, we need to disable the default header:
```

**Comment by thomaswangio** [comment]
> This page is for the `next` version. Can you add them in website/versioned_docs/version-4.x instead?

sure!

File: docs/community-libraries-and-navigators.md (hunk 1 above)
Comment (thomaswangio) [reply docs/community-libraries-and-navigators.md]: done!

**Comment by thomaswangio** [comment]
> > This page is for the `next` version. Can you add them in website/versioned_docs/version-4.x instead?
> 
> wait @satya164 can you tell me how to do this? sorry I'm a bit lost, don't see `community-libraries-and-navigators.md` in `website/versioned_docs/version-4.x` ?

File: docs/community-libraries-and-navigators.md [hunk 3]
```diff
@@ -64,15 +64,33 @@ Provides simple HOCs that map react-navigation props to your screen components d
 
 This component implements [iOS large header with grow/shrink on scroll](https://react-navigation.canny.io/feature-requests/p/ios-11-large-header-and-growshrink-on-scroll), made by [@jonsamp](https://github.com/jonsamp). Note that it doesn't handle header animation between screens, it only handles animating the header title on scroll.
 
-To use this component for header in React Navigation, we need to disable the default header:
+headerMode is an option in createStackNavigator's config, not in navigationOptions. I'd document both, headerMode: 'none' to disable globally, and header: null to disable header for one screen.
```
Comment (satya164) [inline docs/community-libraries-and-navigators.md]: Hmm. I was replying to you, not suggesting to add this text to the README.

Comment (thomaswangio) [reply docs/community-libraries-and-navigators.md]: oh oops totally my bad

**Pull request merged**